jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/DateTimePickersAssert.java

```diff
@@ -17,6 +18,31 @@ public DateTimePickersAssert enabled() {
         return this;
     }
 
+    @JDIAction("Assert that picker index {index} is displayed")
+    public DateTimePickersAssert displayed(final int index) {
+        jdiAssert(element().getPickerField(index).isDisplayed(), Matchers.is(true));
+        return this;
+    }
+
+    @JDIAction("Assert that picker {index} has text '{0}'")
```

Comment:
Please update text of @JDIAction with right condition variant. The same for line 39